packages/block-library/src/image/image.js

```diff
@@ -177,6 +178,17 @@ export default function Image( {
 		}
 	}, [ url, prevUrl ] );
 
+	// Set resizable box dimensions to "auto" when we don't have width/height.
+	// This is done to reset inline styles after changing to a different image.
+	useEffect( () => {
+		if ( url && resizableBoxRef.current && ! width && ! height ) {
+			resizableBoxRef.current.updateSize( {
+				width: 'auto',
+				height: 'auto',
+			} );
+		}
+	}, [ url, width, height ] );
```

Comment:
Why is this being called in a `useEffect` instead of in `updateImage` where the attributes are reset?

Reply:
Or better yet, could we just set the `width` and `height` of the `ResizableBox` component to be `'auto'` when `attributes.width` and `attributes.height` are `undefined`?

```jsx
<ResizableBox
  size={ {
    width: width ?? 'auto',
    height: height ?? 'auto',
  } }
  ...
/>
```

Reply:
Hi, @ajlende

Thanks for the review.

> Why is this being called in a useEffect instead of in updateImage where the attributes are reset?

The attributes are reset in `onSelectImage` and `updateImage`. Using the `useEffect` seemed more optimal at the time.

> Or better yet, could we just set the width and height of the ResizableBox component to be 'auto' when attributes.width and attributes.height are undefined?

Thanks for suggesting a clearer approach 🙇  This is the reason I love reviews.